packages/manager/src/features/Linodes/LinodeCreate/Summary/utilities.ts

```diff
@@ -25,18 +34,75 @@ export const getLinodePrice = (options: LinodePriceOptions) => {
   }
 
   if (isCluster) {
-    const numberOfNodes = Number(clusterSize);
+    let totalClusterSize = Number(clusterSize);
+    let clusterTotalMonthlyPrice = 0;
+    let clusterTotalHourlyPrice = 0;
+
+    if (hasClusterData) {
+      // cluster monthly
+      clusterData.forEach((item) => {
+        const price = getLinodeRegionPrice(item.typeData, regionId);
+        const sizeMultiplier = parseInt(item.size ?? '0', 10);
+        clusterTotalMonthlyPrice += (price?.monthly ?? 0) * sizeMultiplier;
+      });
+
+      // cluster hourly
+      clusterData.forEach((item) => {
+        const price = getLinodeRegionPrice(item.typeData, regionId);
+        const sizeMultiplier = parseInt(item.size ?? '0', 10);
+        clusterTotalHourlyPrice += (price?.hourly ?? 0) * sizeMultiplier;
+      });
 
+      // total cluster size
+      clusterData.forEach((item) => {
+        const sizeMultiplier = parseInt(item.size ?? '0', 10);
+        totalClusterSize += sizeMultiplier;
+      });
+    }
```

Comment:
Can this be simplified to something like:
```suggestion
    let totalClusterSize = Number(clusterSize);
    let clusterTotalMonthlyPrice = 0;
    let clusterTotalHourlyPrice = 0;

    if (hasClusterData) {
      clusterData.forEach((item) => {
        const price = getLinodeRegionPrice(item.typeData, regionId);
        const numberOfNodes = parseInt(item.size ?? '0', 10);
        clusterTotalMonthlyPrice += (price?.monthly ?? 0) * numberOfNodes;
        clusterTotalHourlyPrice += (price?.hourly ?? 0) * numberOfNodes;
        totalClusterSize += numberOfNodes;
      });
    }
```

I seem to be getting a different total price with this code, but I think this produces the correct total pricing? Check behind me!